iofile/listFiles.go

```diff
@@ -1,25 +1,46 @@
 package iofile
```

Comment:
`iofile` means what?

Reply:
I used iofile as input/output file as I was performing all file related activities in this package.

I could change it to something more in line with the expected nomenclature.

Reply:
Standard practise is to call as `io`.